docs/en/ingest-management/overview.asciidoc

```diff
@@ -5,86 +5,47 @@
 [[elastic-agent]]
 == {agent}
 
-{agent} is a single, unified way to add monitoring for logs, metrics, and
-other types of data to each host. A single agent makes it easier and faster
-to deploy monitoring across your infrastructure. The agent's single, unified
-policy makes it easier to add integrations for new data sources.
+//TODO: We need to converge on a single definition for this product because
+//we describe it in many different ways in these docs and others. I've made
+//some minor edits to this description, but it still needs work.
```

Comment:
I think we should come up with standard short descriptions for all the products we work on and sync with PMs and marketing so that we have a standard description. Glossary definitions are a good start, but they fall short of the amount of detail we need for introductory mats. It must be really weird for users to see all the ways we describe the same thing across different docs.